tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1495,21 +1498,27 @@ public:
    /// the stats for all named filters in the chain section between the original
    /// `TDataFrame` and that node (included). Stats are printed in the same
    /// order as the named filters have been added to the graph.
-   void Report()
+   TCutFlowReport Report()
```

Comment:
I think it would be nice if we took a flag that users can toggle to decide whether they want the cuts printed on screen or not.